Source/Core/Core/HW/WiimoteEmu/Extension/Shinkansen.cpp

```diff
@@ -49,7 +49,7 @@ Shinkansen::Shinkansen() : Extension3rdParty("Shinkansen", _trans("Shinkansen Co
   m_levers->AddInput(Translatability::Translate, _trans("R"));
 
   // i18n: Noun. This is used as a label for a control group that represents lights on controllers.
-  groups.emplace_back(m_led = new ControllerEmu::ControlGroup(_trans("Light")));
+  groups.emplace_back(m_led = new ControllerEmu::ControlGroup(_trans("Lights")));
```

Comment:
fyi, there is only one light

Reply:
Yeah. Although it's a group of light controls, there's actually only one light control in it...

The thing is, I'm actually using this to try and resolve a different problem: That this string is identical to the "Light" element from Skylanders. Maybe it would be better to use a context, but I don't know if we have that set up in a way that actually works.

Reply:
This also breaks compatibility with existing configs, but basically no one is using this emulated Shinkansen.

Reply:
That is something I'd rather avoid, considering how easy it is to avoid.